src/enrichers/people-verified-id.ts

```diff
@@ -0,0 +1,20 @@
+import { PEOPLE_VERIFIED_LS_ENTRY } from '../utils/consts'
+import { Enricher, EventBus } from '../types'
+import { WrappedReadOnlyStorageHandler } from '../handlers/storage-handler'
+
+type Input = { peopleVerifiedId?: string }
+type Output = { peopleVerifiedId?: string }
+
+export function enrichPeopleVerifiedId(storageHandler: WrappedReadOnlyStorageHandler, eventBus: EventBus): Enricher<Input, Output> {
+  return state => {
+    try {
+      return {
+        ...state,
+        peopleVerifiedId: state.peopleVerifiedId || storageHandler.getDataFromLocalStorage(PEOPLE_VERIFIED_LS_ENTRY) || undefined
```

Comment:
This should probably use the cache / access the same record as the live-connect-id enricher, but I don't want to change now